README.md

```diff
@@ -15,10 +13,168 @@
 
 -->
 
----
+# Amplify UI 2.0
+
+Amplify UI 2.0 is an open-source UI library with cloud-connected components that are endlessly customizable, accessible, and can integrate into _any_ application. Amplify UI consists of:
+
+1. Connected components that simplify complex cloud-connected workflows, like Authenticator.
```

Comment:
Thoughts on this this being cloud platform specific (i.e. Amplify-connected)?      

Something like "Connected components that simplify complex Amplify workflows, like Authenticator."?

I'm thinking that if a customer lands on this repo and has never heard of Amplify then "cloud connected" could mean many different things, especially for those that are more frontend focused.